### DISCUSSION TITLE
New feature

### DISCUSSION THREAD

**Discussion by Sloom128** [MissingCore/Music#84] [Ideas]
A new idea would be to. Make the spinning disc another progress bar (timeline) controller for precise needs
![PSX_20240911_230131](https://github.com/user-attachments/assets/fa714432-53e9-4b05-b6b6-f6bc24e53f98)

**Comment by cyanChill** [comment]
When implementing this feature, I mainly intended it to be a gimmick. With the next major update, the current implementation is just going to be replaced with the artwork as the original design didn't look too good in light mode. I do plan on adding an alterative choice (an improved disc design) in a future update. ~~In addition, I currently don't see the need for it as with the current seekbar, you can scrub per second given the intended use case (audio files will generally be short enough for this to occur) and will probably be a complex thing to implement (ie: how much time do we want to traverse given a set motion and figuring out the underlying math to do this).~~

**Reply by Sloom128** [reply]
Take your time.

**Comment by cyanChill** [comment]
If you're using the APK version of the app, you can try out the new design with the [`v2.1.0-rc.1`](https://github.com/MissingCore/Music/releases/tag/v2.1.0-rc.1) release. In addition, we ended up giving the vinyl the ability to act as a seekbar, with a full revolution being 24s.
